From 7aa96930ad47286f9ee778219671fb5e76781817 Mon Sep 17 00:00:00 2001
From: Mika Laitio <lamikr@gmail.com>
Date: Fri, 16 Aug 2024 14:22:42 -0700
Subject: [PATCH 7/7] add strace about files downloaded

- helps seeing from where where the pytorch downloads the
  audio files and where it saves them when running example
  applications.

Signed-off-by: Mika Laitio <lamikr@gmail.com>
---
 src/torchaudio/utils/download.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/torchaudio/utils/download.py b/src/torchaudio/utils/download.py
index 2081877d..3326aac5 100644
--- a/src/torchaudio/utils/download.py
+++ b/src/torchaudio/utils/download.py
@@ -18,6 +18,7 @@ def _get_local_path(key):
 
 def _download(key, path, progress):
     url = f"https://download.pytorch.org/torchaudio/{key}"
+    print("url: " + str(url))
     download_url_to_file(url, path, progress=progress)
 
 
@@ -69,6 +70,7 @@ def download_asset(
     path = path or _get_local_path(key)
 
     if path.exists():
+        print("The local file (%s) exists. Skipping the download: " + str(path))
         _LG.info("The local file (%s) exists. Skipping the download.", path)
     else:
         _LG.info("Downloading %s to %s", key, path)
-- 
2.46.0

